app/src/stores/client.ts

```diff
@@ -130,7 +130,7 @@ export const useClientStore = defineStore({
     async unarchiveOffer(updateOffer: PatchOffer) {
       this.loadingState = LoadingState.show('Archiving Offer...')
       try {
-        updateOffer.state = OfferState.paused
+        updateOffer.state = OfferState.PAUSED
```

Comment:
Please change it back to all lowercase. I don't think it is working in uppercase, since in `OfferState` it is set to lowercase.